uberfire-project/uberfire-project-backend/src/main/java/org/guvnor/common/services/project/backend/server/utils/configuration/ConfigurationKey.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.guvnor.common.services.project.backend.server.utils.configuration;
+
+/**
+ * Keys use to configure the Maven compiler in the kie-wb-common
+ *
+ * <p>
+ *  COMPILER jdt or javac
+ *  SOURCE_VERSION  configured with jvm source code version
+ *  TARGET_VERSION  configured with jvm target version
+ *  <p>
+ *  FAIL_ON_ERROR configured with false to continue the build on the correct classes and skip the build of classes with errors
+ *  MAVEN_COMPILER_PLUGIN_GROUP  configured with default maven compiler group to disabled it
```

Comment:
I don't understand this comment `configured with default maven compiler group to disabled it` - what do you mean by it? What do I need to disable and why? When would it make sense to enable it?